<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKBaseListWidget</class>
 <widget class="QWidget" name="SAKBaseListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="4">
    <widget class="QPushButton" name="importPushButton">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="8">
    <widget class="QListWidget" name="itemListWidget"/>
   </item>
   <item row="1" column="5">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="infoLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="addPushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="clearPushButton">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="exportPushButton">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="deletePushButton">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="forbidAllItemsCheckBox">
     <property name="text">
      <string>Forbid all items</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
